extensions/amp-ad-custom/0.1/amp-ad-custom.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {AmpAdTemplate} from '../../amp-a4a/0.1/amp-ad-template';
```

Comment:
should we just move AmpAdTemplate here?

Reply:
Do you mean move the whole module into this file, or just move the module into this extension?

Reply:
whole module to this file. basically merge the 2 files.

Reply:
Done.